modules/tensor_mechanics/src/actions/TensorMechanicsAxisymmetricRZAction.C

```diff
@@ -0,0 +1,68 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "TensorMechanicsAxisymmetricRZAction.h"
+
+#include "Factory.h"
+#include "FEProblem.h"
+#include "Parser.h"
+
+template<>
+InputParameters validParams<TensorMechanicsAxisymmetricRZAction>()
+{
+  InputParameters params = validParams<Action>();
+  params.addClassDescription("Set up stress divergence kernels");
+  params.addRequiredParam<std::vector < NonlinearVariableName>  >("displacements", "The nonlinear displacement variables for the problem");
```

Comment:
There is a bit too much whitespace here. Every space before the ```(``` can go, except the one between the ```> >```.